docker/README.md

```diff
@@ -0,0 +1,105 @@
+# Data Privacy Toolkit - CLI
+
+This project contains the command line interface (CLI), and the corresponding Docker image, of the Data Privacy Toolkit (DPT).
```

Comment:
This project contains the command line interface (CLI) and the corresponding Docker image of the Data Privacy Toolkit (DPT)